1-mandatory/1-writers.js

```diff
@@ -73,6 +83,12 @@ Exercise 3:
 
 function logAliveWritersInTheirForties() {
   // write your code here
+  for (let i = 0; i < writers.length; i++) {
+    const writer = writers[i];
+    if (writer.age >= 40 && writer.age < 50 && writer.alive){
+     console.log(`Hi, my name is ${writer.firstName} ${writer.lastName}. I am ${writer.age} years old.`);
+    }
+}
 }
```

Comment:
Well done > I really recommend you to use the other way for loop =>  for(let ... of ....) which is now more popular is very easy to use